### PR TITLE
Use check_disk thresholds from pillar data

### PR DESCRIPTION
**Pull request by paulmathews**
In some environments, the instance's given disk size is such that it is
allowed to expand and fill the host's disk, pushing the host disk over the
specified thresholds. This allows modification of these thresholds via pillar
data.

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: linux/meta/sensu.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ check:
     - {{ system.name|replace('.', '-') }}-{{ system.domain|replace('.', '-') }}
 {%- endif %}
   local_linux_storage_disk_usage:
-    command: "PATH=$PATH:/usr/lib64/nagios/plugins:/usr/lib/nagios/plugins check_disk -w 15% -c 5% -p / -p /var -p /usr -p /tmp -p /var/log"
+    command: "PATH=$PATH:/usr/lib64/nagios/plugins:/usr/lib/nagios/plugins check_disk -w" {{ monitoring.disk.get('warn', '15%') }} " -c " {{ monitoring.disk.get('critical', '5%') }}  " -p / -p /var -p /usr -p /tmp -p /var/log"
```
Comment (fpytloun) [inline linux/meta/sensu.yml]: This will not work if monitoring or monitoring.disk is not defined.
What's monitoring pillar structure anyway? Was this agreed by @cznewt to be used this way?

Comment (geekinutah) [inline linux/meta/sensu.yml]: Hey Filip, looks like this will use the default if the pillar data is not there. I don't see a problem here.

Comment (fpytloun) [inline linux/meta/sensu.yml]: No, it will not work at all:
1. monitoring is not defined (seems you didn't test this change), for what you are trying to achieve, this should be `pillar.monitoring` or `system.monitoring`, depends on agreed metadata structure for overriding defaults - @cznewt needs to give feedback here
2. ok, if `monitoring` is defined, you need to check for child structure as well, so it should be `monitoring.get('disk', {}).get('warn', '15%')`

Comment (cznewt) [inline linux/meta/sensu.yml]: @geekinutah hello, do you have some example of the monitoring data, some sample metadata should be added to the readme/tests to display the model for this feature. this is something that needs to be handled on all formulas.

Comment (geekinutah) [inline linux/meta/sensu.yml]: Please close this, work will need to be done in another PR based on:
https://github.com/salt-formulas/salt-formula-linux/pull/89

**Comment by epcim** [comment]
closed in favor of #89
